### PR TITLE
fix(cli): ca and certs are not included in loading db from CLI

### PR DESCRIPTION
**Pull request by ihexxa**
### Changes
- [x] fix: ca and certs are not included in loading db
- [x] Added a test for cert db loading
- [x] Added a test for settings db loading

Ref: INS-4598

### CODE REVIEW

**Review by filfreire — DISMISSED** [review]

**Review by ryan-willis — DISMISSED** [review]

**Review by jackkav — COMMENTED** [review]
Is this just a plumbing PR or is there a case that works now that didn't before? The word fix implies something wasn't working before, if that is the case there probably could be a test case to show what it was.

**Comment by ihexxa** [comment]
Let me take a look and cover it, originally it was a quick fix but as more changes will be introduced, setting it to draft.

**Comment by ihexxa** [comment]
Added tests with fake db, certs. The cert test will load the fake cert although it will finally fail, which should be ok.
And the settings test will load settings db and doesn't follow the 302 redirection as `allowRedirect` is disabled in settings. I choose testing this option because almost any setting potentially could affect existing tests' behavior.

File: packages/insomnia-inso/src/db/fixtures/certs/fake_key.pem [hunk 1]
```diff
@@ -0,0 +1 @@
+fake key content
```
Comment (filfreire) [inline packages/insomnia-inso/src/db/fixtures/certs/fake_key.pem:1]: for a future PR pass I recommend we also try with a working cert

File: packages/insomnia-inso/src/cli.test.ts [hunk 2]
```diff
@@ -126,6 +126,22 @@ describe('inso dev bundle', () => {
...
     });
+
+    it('send request with client cert and key', async () => {
+      const input = '$PWD/packages/insomnia-inso/bin/inso run collection -w packages/insomnia-inso/src/db/fixtures/nedb --requestNamePattern "withCertAndCA" --verbose "Insomnia Designer"';
+      const result = await runCliFromRoot(input);
+      if (result.code !== 0) {
+        console.log(result);
+      }
+      expect(result.stdout).toContain('Adding SSL PEM certificate');
+      expect(result.stdout).toContain('Adding SSL KEY certificate');
```
Comment (filfreire) [inline packages/insomnia-inso/src/cli.test.ts:137]: 👍

**Review by filfreire — APPROVED** [review]

File: packages/insomnia-inso/src/db/fixtures/certs/fake_key.pem (hunk 1 above)
Comment (ihexxa) [reply packages/insomnia-inso/src/db/fixtures/certs/fake_key.pem:1]: +1 originally I added one but the key was blamed by the github bot above, probably we could try with cert. 🥹

**Comment by jackkav** [comment]
It's easier to fix this test later if you isolate the case using a yml input rather than nedb one and would fit the convention of the other tests in this group you're adding to.

File: packages/insomnia-inso/src/db/fixtures/certs/fake_key.pem (hunk 1 above)
Comment (jackkav) [reply packages/insomnia-inso/src/db/fixtures/certs/fake_key.pem:1]: Can't you ignore the GitHub bot? A happy test is more valuable than a sad one.

**Comment by ihexxa** [comment]
@jackkav I agree with your points about testing tls connection and tests with yml, as this PR mainly focuses on fixing data loading so I propose to extend it in the next.

**Pull request merged**